lib/medtronic/processData.js

```diff
@@ -373,6 +387,36 @@ var buildBolusRecords = function (records) {
   return postrecords;
 };
 
+function buildBGRecords(records) {
+
+  var bgRecords = filterHistory([RECORD_TYPES.BG_RECEIVED], records);
+  var postrecords = [];
+
+  bgRecords.forEach(function (bgEntry) {
+
+    console.log('smbg:',bgEntry.jsDate, common.bytes2hex(bgEntry.head), common.bytes2hex(bgEntry.date), common.bytes2hex(bgEntry.body));
```

Comment:
Indeed. Will be changing all `console.log` statements to `debug` statements and wrap some in `if(debug)` statements at a future stage.